CHANGES.md

```diff
@@ -1,3 +1,16 @@
+## edge-18.9.2
+
+* **New** _edge_ and _stable_ release channels
+* Web UI
+  * **Improved** Tap & Top UIs with better layout and linking
+* CLI
+  * **Improved** `check --pre` command verifies the caller has sufficient
+    permissions to install Linkerd
+  * **Improved** `check` command verifies that Prometheus has data for proxied
```

Comment:
i think this was captured in the previous release as `New linkerd check now validates Linkerd proxy versions and readiness` ... emphasis on `readiness`, but i suppose we could include it here if we want to call it out specifically.

Reply:
This refers to the check that was added in #1635, which wasn't included in the previous release.